src/test/java/browsertests/day3.java

```diff
@@ -0,0 +1,50 @@
+package browsertests;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class day3 {
```

Comment:
Classes should start with capital letters and not include numbers.

Consider naming this to `DayThree`